common/flatpak-dir.c

```diff
@@ -7550,13 +7576,19 @@ flatpak_dir_remote_list_refs (FlatpakDir       *self,
   return TRUE;
 }
 
+typedef enum {
+  FIND_MATCHING_REFS_FLAGS_NONE = 0,
+  FIND_MATCHING_REFS_FLAGS_KEEP_REMOTE = (1 << 0)
```

Comment:
Nitpick: Add a trailing comma on this one to avoid diff noise in future if adding more enum members.